python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -137,6 +137,38 @@
     db.Column("create_timestamp", db.DateTime, server_default=get_current_timestamp()),
 )
 
+AssetCheckExecutionsTable = db.Table(
+    "asset_check_executions",
```

Comment:
nit: are there things other than executions?  what about just `asset_checks`?

Reply:
I don't think there will be anything beyond executions... this is roughly analogous to the runs table. Could call it asset_check_runs but I thought that might be confusing since one Run might execute many asset checks